web/src/components/Note.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+
+import { NoteIcon } from '../assets'
+import Icon from './base/Icon'
+
+const NoteContainer = styled.div<{ visible: boolean }>`
+  display: flex;
+  background-color: ${props => props.theme.colors.themeColor};
+  padding: 12px;
+  gap: 12px;
+  align-items: center;
+  visibility: ${props => !props.visible && 'hidden'};
+`
+
+const NoteText = styled.span`
+  font-size: ${props => props.theme.fonts.decorativeFontSizeSmall};
+`
+
+type NoteProps = {
+  text: string
+  visible: boolean
+}
+
+const Note = ({ text, visible }: NoteProps): ReactElement => (
+  <NoteContainer visible={visible}>
```

Comment:
Same question as in native: I'd prefer it if the button moves up when the note disappears. I also checked back with @hauf-toni regarding this. Why did you prefer hiding it?

Reply:
That was the way it was implemented bevor. It was at least partly on my initiative though since I don't like jumping layouts. If I want to click the button and then it moves up that is annoying to me. However, if UI/UX says thats how it should be done we can easily do that.

Reply:
Created https://github.com/digitalfabrik/integreat-app/issues/2597.